pwiz_tools/Skyline/CommandArgUsage.resx

```diff
@@ -724,4 +724,7 @@ greater peak area.</value>
   <data name="_refine_max_precursor_only" xml:space="preserve">
     <value>Only the precursor charge state with the maximum peak area will be retained.</value>
   </data>
+  <data name="_exp_order_by_mz" xml:space="preserve">
+    <value>Order exported method by m/z, ascending.</value>
```

Comment:
Add "for optimized quadrupole switching" for a better explanation. This is the worst kind of documentation. It really just restates what you can read in the name other than "ascending". We could also add "(precursor m/z, then product m/z when applicable)" to be even more descriptive about what this is going to do.